src/report/utility-reports/hello-world.scm

```diff
@@ -354,9 +354,9 @@ or extending existing reports.")))
 new, totally cool report, consult the mailing list %s.")
           (gnc:html-markup-anchor 
            "mailto:gnucash-devel@gnucash.org"
-           (gnc:html-markup-tt "gnucash-devel@gnucash.org")))
-         (_ "For details on subscribing to that list, see &lt;http://www.gnucash.org/&gt;.")
-         (_ "You can learn more about writing scheme at &lt;http://www.scheme.com/tspl2d/&gt;."))
+           (gnc:html-markup-tt "gnucash-devel@gnucash.org"))
+          (_ "For details on subscribing to that list, see &lt;http://www.gnucash.org/&gt;. ")
+          (_ "You can learn more about writing scheme at &lt;http://www.scheme.com/tspl2d/&gt;.")))
```

Comment:
Perhaps another approach to fixing this would be to add a line break after the a href ?